packs/ansible/actions/galaxy.install.yaml

```diff
@@ -0,0 +1,52 @@
+---
+name: "galaxy.install"
+description: "Download & Install role from ansible galaxy"
+runner_type: "local-shell-script"
+entry_point: "ansible_galaxy.py"
+enabled: true
+parameters:
+  kwarg_op:
+    description: "Lock operator type to '--'"
+    type: string
+    default: "--"
+    immutable: true
+  sudo:
+    description: "Lock sudo, the behavior is controlled by ansible 'become_' options"
+    type: boolean
+    default: true
+    immutable: true
+  timeout:
+    description: "Action timeout in seconds. Action will get killed if it doesn't finish in timeout seconds"
+    type: integer
+    default: 300
+  cwd:
+    description: "Working directory where the command will be executed in"
+    type: string
+  action:
+    description: "Action to use"
+    type: string
+    position: 0
+    default: install
+    immutable: true
+  roles:
```

Comment:
Since my implementation is just simple wrapper, this part should be handled by Ansible itself and it gives good enough message: 

Running with both roles and role-file: 
```sh
./ansible-galaxy install abc.de -r file.txt
st2 run ansible.galaxy.install roles=abc role-file=cde.txt
```
Output: 
```sh
status: failed
result:
{
    "failed": true,
    "stderr": "Executed command "/opt/stackstorm/virtualenvs/ansible/bin/ansible-galaxy --role-file=cde.txt install abc"
",
    "return_code": 1,
    "succeeded": false,
    "stdout": "Usage: ansible-galaxy install [options] [-r FILE | role_name(s)[,version] | scm+role_repo_url[,version] | tar_file(s)]

Options:
  -h, --help            show this help message and exit
  -i, --ignore-errors   Ignore errors and continue with the next specified
                        role.
  -n, --no-deps         Don't download roles listed as dependencies
  -r ROLE_FILE, --role-file=ROLE_FILE
                        A file containing a list of roles to be imported
  -p ROLES_PATH, --roles-path=ROLES_PATH
                        The path to the directory containing your roles. The
                        default is the roles_path configured in your
                        ansible.cfg file (/etc/ansible/roles if not
                        configured)
  -s API_SERVER, --server=API_SERVER
                        The API server destination
  -f, --force           Force overwriting an existing role

See 'ansible-galaxy <command> --help' for more information on a specific command.

- please specify a user/role name, or a roles file, but not both
"
}
```